src/web/og.mli

```diff
@@ -0,0 +1,48 @@
+type url = { name : string }
+type locale = { language : string; territory : string }
```

Comment:
A few comment to explain what this is about, + a link to the wikipedia page of open graph (if there is one), please.